### PR TITLE
Merged realms into single "prices" realm. Refactoring.

### PR DESCRIPTION
**Pull request by polonzer**
https://lykkex.atlassian.net/browse/LWDEV-5545

### CODE REVIEW

File: src/Lykke.Frontend.WampHost/Modules/BalancesModule.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using Autofac;
+using Lykke.Frontend.WampHost.Core.Services;
+using Lykke.Frontend.WampHost.Core.Settings;
+using Lykke.Frontend.WampHost.Services.Balances;
+
+namespace Lykke.Frontend.WampHost.Modules
+{
+    public class BalancesModule : Module
```
Comment (MikhailGerasimov) [inline src/Lykke.Frontend.WampHost/Modules/BalancesModule.cs]: Can be internal

File: src/Lykke.Frontend.WampHost/Modules/CandlesModule.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿using Autofac;
+using Lykke.Frontend.WampHost.Core.Domain;
+using Lykke.Frontend.WampHost.Core.Services;
+using Lykke.Frontend.WampHost.Core.Services.Candles;
+using Lykke.Frontend.WampHost.Core.Settings;
+using Lykke.Frontend.WampHost.Services.Candles;
+
+namespace Lykke.Frontend.WampHost.Modules
+{
+    public class CandlesModule : Module
```
Comment (MikhailGerasimov) [inline src/Lykke.Frontend.WampHost/Modules/CandlesModule.cs]: Can be internal

File: src/Lykke.Frontend.WampHost/Modules/QuotesModule.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+using Lykke.Frontend.WampHost.Core.Services;
+using Lykke.Frontend.WampHost.Core.Services.Quotes;
+using Lykke.Frontend.WampHost.Core.Settings;
+using Lykke.Frontend.WampHost.Services.Quotes;
+using Lykke.Frontend.WampHost.Services.Quotes.Mt;
+using Lykke.Frontend.WampHost.Services.Quotes.Spot;
+
+namespace Lykke.Frontend.WampHost.Modules
+{
+    public class QuotesModule : Module
```
Comment (MikhailGerasimov) [inline src/Lykke.Frontend.WampHost/Modules/QuotesModule.cs]: Can be internal

Comment (polonzer) [reply src/Lykke.Frontend.WampHost/Modules/QuotesModule.cs]: Yep, we should not use 'public' so much :)

**Pull request merged**